repo2docker/app.py

```diff
@@ -474,6 +475,19 @@ def _get_free_port(self):
         return port
 
     def start(self):
+        # Check if r2d can connect to docker daemon
+        if self.build:
+            try:
+                client = docker.APIClient(version='auto',
+                                          **kwargs_from_env())
+                del client
+            except DockerException as e:
+                print("Docker client initialization error. Check if docker is running in the host.")
```

Comment:
"Check if docker is running in the host." -> "Check if docker is running on the host."

Reply:
Done